### ISSUE TITLE
Cases failed to run at https://js.tensorflow.org/api/latest/

### ISSUE THREAD

**Issue by axinging** [tensorflow/tfjs#7289]
**Case 1: tf.MathBackendWebGL** 
Search "const customBackend = new tf.MathBackendWebGL(customCanvas);", scroll to the case, click run. It complains:
```
An error occured on line: 10
tf.MathBackendWebGL is not a constructor
```

**Case 2: createGPUBufferFromData** 
Search "createGPUBufferFromData", and run it under chrome canary(with  --enable-unsafe-webgpu), it complains:
```
An error occured on line: 17
Cannot read properties of undefined (reading 'createBuffer')
```

For case 2, it only runs in webgpu backend, possibly we need below two steps to fix it:
1, set webgpu backend:
```
await tf.setBackend('webgpu');
await tf.ready();
```
2, add tfjs webgpu backend:
Either support 'import '@tensorflow/tfjs-backend-webgpu';'
Or add: <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgpu"></script>

**Comment by Linchenn** [comment]
@pyu10055 @mattsoulanille 

TLDR for the issue: A code snippet on our API website depends on tfjs-backend-webgpu, because the feature is for WebGPU backend. Since we haven't fully supported/announced WebGPU, we could either 1. hide the feature from the API website, or 2. Add `<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgpu"></script>`. What do you think?

**Comment by mattsoulanille** [comment]
I think adding webgpu to the API site is a good idea, however, we should sign up for an [origin trial](https://developer.chrome.com/en/docs/web-platform/webgpu/#register-for-ot). Otherwise, it won't work, even if the browser supports it.

We may also want to note that these APIs require webgpu if we haven't already.

**Comment by Linchenn** [comment]
@qjia7 @axinging 
FYI, will add the latest version of tfjs-backend-webgpu to our API website, https://github.com/tensorflow/tfjs-website/pull/488.
